database/seed.js

```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+const db = require('./index.js');
+mongoose.connect('mongodb://localhost/opentable');
+
+const cuisines = ['American', 'Italian', 'Steakhouse', 'Seafood', 'French', 'Indian', 'Mexican', 'Japanese', 'Chinese', 'German', 'Spanish', 'Fusion/Eclectic', 'Barbecue', 'Greek', 'Grill', 'Comfort Food', 'Turkish', 'International', 'Thai', 'Tex-Mex', 'Bistro', 'Bar/Lounge', 'Californian'];
+const styles = ['Casual Elegant', 'Casual Dining', 'Fine Dining', 'Elegant Dining'];
+const mainTags = ['Fit For Foodies', 'Neighborhood Gem', 'Special Occasion', 'Tasting Menu', 'Quiet Conversation', 'Creative Cuisine', 'Casual', 'Romantic', 'Local Ingredients', 'Notable Wine List', 'Authentic', 'Fun', 'Vibrant Bar Scene', 'Good For A Date'];
+const additionalTags = ['Bar Dining', 'Bar/Lounge', 'Beer', 'Cocktails', 'Corkage Fee', 'Counter Seating', 'Farm to Table', 'Fireplace', 'Full Bar', 'Gluten-free Menu', 'Happy Hour', 'Late Night', 'Non-Smoking', 'Outdoor dining', 'Patio/Outdoor Dining', 'Weekend Brunch', 'Wheelchair Access', 'Wine'];
+const payments = ['AMEX', 'Discover', 'MasterCard', 'Visa'];
+const dressStyles = ['Casual Dress', 'Business Casual', 'Smart Casual', 'Jacket Preferred'];
+const neighborhoods = ['Parkton', 'Bowdle', 'Ilchester', 'Lanesville', 'Pin Oak Acres', 'Lennox', 'Metcalfe', 'Lynden', 'New Cumberland', 'New Troy', 'Mapleton'];
+const days = 'Monday - ' + faker.random.arrayElement(['Friday', 'Saturday', 'Sunday']);
+const times = faker.random.arrayElement(['7 AM - ', '8 AM - ', '9 AM - ']) + faker.random.arrayElement(['7 PM', '8 PM', '9 PM']);
+
+const percentBool = (number) => {
+  if (Math.random() < number/100) {
```

Comment:
This part kind of confuses me! Since Math.random() gives you a number from 0 to 1, it will always be smaller than number/100 unless number is greater than 100. Just checking!

Reply:
This function takes a number between 1 and 100 and converts it to a decimal percentage (/100). Since Math.random() is a random number between 0 and 1, we would expect Math.random() would be less than number/100 that amount of the time. If 'number' is 100, 100/100 = 1, and we would expect Math.random() to be less than 1 100% of the time. If 'number' is 0, 0/100 = 0, and we would expect Math.random() to be less than 0 0% of the time. I made this function so that I could add fields to some restaurant details pages but not others, because some types of fields appear more consistently than others. For example, restaurants might have a 'Chef' section 80% of the time, but only have 'Private Parties' 40% of the time